src/app/activity/bounty/Create.ts

```diff
@@ -1,145 +1,187 @@
+        const owedTo = await DiscordUtils.getGuildMemberFromUserId(createRequest.owedTo, createRequest.guildId);
+        newBounty = await createDbHandler(
+            createRequest,
+            null,
+            null,
+            null,
+            guildMember,
+            owedTo);
+    }
 
-    const newBounty = await createDbHandler(
-        createRequest,
-        description,
-        criteria,
-        dueAt,
-        guildMember);
+    Log.info(`user ${guildMember.user.tag} inserted bounty into db`);
 
+    let bountyCard: MessageOptions;
 
-    Log.info(`user ${guildMember.user.tag} inserted bounty into db`);
-    let bountyPreview: MessageOptions = {
-        embeds: [{
-            title: await BountyUtils.createPublicTitle(newBounty),
-            url: (process.env.BOUNTY_BOARD_URL + newBounty._id),
-            author: {
-                icon_url: guildMember.user.avatarURL(),
-                name: `${newBounty.createdBy.discordHandle}: ${guildId}`,
-            },
-            description: newBounty.description,
-            fields: [
-                // TODO: figure out a way to explicitly match order with BountyEmbedFields
-                // static bountyId = 0;
-                // static criteria = 1;
-                // static reward = 2;
-                // static status = 3;
-                // static deadline = 4;
-                // static createdBy = 5;
-                { name: 'Bounty Id', value: newBounty._id.toString(), inline: false },
-                { name: 'Criteria', value: newBounty.criteria.toString() },
-                { name: 'Reward', value: newBounty.reward.amount + ' ' + newBounty.reward.currency, inline: true },
-                { name: 'Status', value: BountyStatus.open, inline: true },
-                { name: 'Deadline', value: BountyUtils.formatDisplayDate(newBounty.dueAt), inline: true },
-                { name: 'Created by', value: newBounty.createdBy.discordHandle.toString(), inline: true },
-            ],
-            timestamp: new Date().getTime(),
-            footer: {
-                text: '👍 - publish | ❌ - delete | Please reply within 60 minutes',
-            },
-        }],
-    };
+    if (createRequest.isIOU) {
+        let bountyCard: MessageOptions = {
+            embeds: [{
+                title: await BountyUtils.createPublicTitle(newBounty),
+                url: (process.env.BOUNTY_BOARD_URL + newBounty._id),
+                author: {
+                    icon_url: guildMember.user.avatarURL(),
+                    name: `${newBounty.createdBy.discordHandle}: ${guildId}`,
+                },
+                description: newBounty.description,
+                fields: [
+                    { name: 'IOU Id', value: newBounty._id.toString(), inline: false },
+                    { name: 'Reward', value: newBounty.reward.amount + ' ' + newBounty.reward.currency, inline: true },
+                    { name: 'Status', value: BountyStatus.open, inline: true },
```

Comment:
I think with IOUs we want to guide users to understand that they can set IOUs to paid or to unpaid. Deletion is secondary (and the rest of the bounty lifecycle doesn't really appy), so I don't think there's a lot of value in presenting `Status`. What do you think of changing this to paid status?

Reply:
I think using the underlying status as open, completed, or deleted as fine, if you can just comment what's going on and why. 

Paid status is unrelated to those statuses. For IOUs, I'd like users to only see the paid status (while keeping functionality for delete)

Reply:
Done